### ISSUE TITLE
Loadouts not transferring properly

### ISSUE THREAD

**Issue by inexorableAce** [DestinyItemManager/DIM#2466]
When trying to equip loadout, all items (including subclass) receive an item not found error. Despite the items already being in the inventory. Tested in iOS with multiple loadouts, old and new. The items can be equipped one at a time.

**Comment by doktor5000** [comment]
Also happens when using Maximize Power, and when all the items are in the inventory already.

**Comment by wmxp** [comment]
Having the same issue. Red box pops up: "Bungie.net message: The requested item was not found", despite correctly moving said item. Happens only when moving from character to character, not character to vault - all methods effected. (Loadout, drag and drop, click-store).   

Is causing problems with items not being moved when using large loadouts that shunt around a lot of gear. Especially has problems with applying the default equip slot for each loadout, which often fails now. Seems like the app is sending commands to Bungie's servers faster than it wants to process them.  

Tested on DIM 4.29.0 via Chrome on Windows 7, Chrome/Firefox on Android 7.1.2 and Chrome/Firefox on iOS 11.

**Comment by h3adache** [comment]
I get this error too. Interestingly, it happens when it has to move items between characters and is coming from an error in `/TransferItem`
```
ErrorCode:1623
ErrorStatus:"DestinyItemNotFound"
Message:"The item requested was not found."
```
But the item is transferred just fine because there's actually 2 calls to transfer the same item.
After that DIM gets "stuck" thinking the transfer fails.

If you drag the item to equip it after that (even if there are multiple errors just pick one of them).
It will correctly call `/EquipItem` and the error goes away for the remaining items.

So to summarize:
Steps:
1. Maximize power on a character without the items to maximize power
2. Observe error
3. Pick one of those items that failed (it should already be on the target character)
4. Drag it to equip it
5. Reattempt Maximize power to properly equip remaining items.

**Comment by 48klocs** [comment]
This should be fixed in beta.

@ me if you continue to see this error.
